app/views/patients/data_entry.html.erb

```diff
@@ -29,6 +29,9 @@
   <%= f.text_field :state, autocomplete: 'off' %>
   <%= f.text_field :county, autocomplete: 'off' %>
 
+   <%= f.select :language, options_for_select(language_options), autocomplete: 'off' %>
+   <%= f.select :voicemail_preference, options_for_select(voicemail_preference_options), autocomplete: 'off' %>
```

Comment:
```suggestion
   <%= f.select :voicemail_preference, options_for_select(voicemail_options), autocomplete: 'off' %>
```

Great thinking on the helper! We actually have a similar one implemented on the Dashboard helper`app/helpers/dashboards_helper.rb:voicemail_options` that we can take advantage of here! That will mean you won't need to new patient helper function, nor the config/locale/en changes, I believe.